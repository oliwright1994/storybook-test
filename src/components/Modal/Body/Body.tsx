import { ModalBody as ChakraModalBody } from '@chakra-ui/core'
import React, { FC } from 'react'

const Body: FC = ({ children }) => {
  return <ChakraModalBody>{children}</ChakraModalBody>
}

export default Body
