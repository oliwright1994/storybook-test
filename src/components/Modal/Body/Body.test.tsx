import React from 'react'
import { renderWithBrightTheme } from 'testing/helpers/themeHelper'

import Body from '.'

describe('<Body />', () => {
  const defaultText = 'Test Text'
  it('should render as expected', () => {
    const { container } = renderWithBrightTheme(<Body>{defaultText}</Body>)

    expect(container).toMatchSnapshot()
  })

  it('should render passed children', () => {
    const { getByText } = renderWithBrightTheme(<Body>{defaultText}</Body>)

    expect(getByText(/test text/i)).toBeTruthy()
  })
})
