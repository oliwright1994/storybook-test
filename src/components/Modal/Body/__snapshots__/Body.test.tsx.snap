// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Body /> should render as expected 1`] = `
<div>
  <div
    class="css-5m10fw"
  >
    Test Text
  </div>
</div>
`;
